src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -62,10 +62,6 @@ public function __construct(float $origin = null, bool $morePrecision = false)
      */
     public function get($id)
     {
-        if (null === $id) {
```

Comment:
Can we add the `string` type hint on `$id`?

Reply:
@jschaedl is adding them among others in #32242